opm/autodiff/BlackoilPropsAdFromDeck.cpp

```diff
@@ -142,6 +143,7 @@ BlackoilPropsAdFromDeck::BlackoilPropsAdFromDeck(const BlackoilPropsAdFromDeck&
     // For data that is dependant on the subgrid we simply allocate space
     // and initialize with obviously bogus numbers.
     cellPvtRegionIdx_.resize(number_of_cells, std::numeric_limits<int>::min());
```

Comment:
I wondered about this a little (even though it is existing and not new code): the bogus init is ok because this will be communicated, correct?